kubernetes-examples/network-topologies/snirouting_tls_in_cluster/overlays/minikube/kroxylicious/server-certificate.yaml

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: server-certificate
+spec:
+  isCA: true
+  commonName: mycluster-proxy.kafka
+  secretName: kroxy-server-key-material
+  privateKey:
+    algorithm: RSA
+    size: 4096
+    # Note the cert-manager default is PKCS1. The tls.key generated won't work with Kroxylicious unless you put
+    # Bouncy Castle in the classpath.  PKCS8 works without the classpath addition.
+    encoding: PKCS8
+  keystores:
+   pkcs12:
+     create: true
+     passwordSecretRef:
+       name: kroxy-server-keystore-password
+       key: storePassword
+  dnsNames:
+    - my-cluster-proxy
+    - my-cluster-proxy-broker-1
+    - my-cluster-proxy-broker-2
+    - my-cluster-proxy-broker-2
```

Comment:
realised that this is only half of the configuration, if we want to enable clients in the `kroxylicious` namespace to connect to `my-cluster-proxy` we would need to add another VirtualCluster. Maybe it just makes the example messier and we should remove these from `dnsNames`. Clients in `kroxylicious` can still access it via `my-cluster-proxy.kroxylicious`.